_dev/src/components/block/card-status.vue

```diff
@@ -65,6 +65,28 @@
         <i class="material-icons">error_outline</i> {{ $t('block.payment-status.limited') }}
       </label>
     </template>
+    <template v-else-if="cardStatus === 'SUSPENDED'">
+      <p v-if="displayLabels">
```

Comment:
This `<template>` has been duplicated 6 times in this file, it is interesting to refacto this in a single component.

It seems the occurences you created here lack the `class="fs-14"`?

Reply:
@v4lux Did you know about `fs-14` class ?
VueJS part is a mystery for me🔮

Reply:
This class is no more needed.